src/ui/wxWidgets/MenuFileHandlers.cpp

```diff
@@ -820,7 +828,7 @@ void PasswordSafeFrame::DoExportText()
           newfile = fd.GetPath().c_str();
           CReport rpt;
 
-          rpt.StartReport(IDSC_RPTEXPORTTEXT, sx_temp.c_str());
+          rpt.StartReport(ExportType::GetReportNameId(), sx_temp.c_str());
           rpt.WriteLine(tostdstring(wxString(_("Exporting database: ")) << towxstring(sx_temp) << _(" to ") << newfile<< wxT("\r\n")));
```

Comment:
Side note: another example of concatenating translatable strings which only works for languages with the same word order as English. This should've been a template string.

Reply:
Added a FIXME comment so as not to forget this.